lib/ReactViews/SidePanel/Branding.jsx

```diff
@@ -18,18 +19,26 @@ const Branding = createReactClass({
       .brandBarElements;
     if (!defined(brandingHtmlElements)) {
       brandingHtmlElements = [
-        '<a target="_blank" href="http://terria.io"><img src="images/terria_logo.png" height="52" title="Version: {{ version }}" /></a>'
+        "",
+        '<a target="_blank" href="http://terria.io"><img src="images/terria_logo.png" height="52" title="Version: {{ version }}" /></a>',
+        ""
```

Comment:
Note I had to change this so that the "default" would be an array of 3 like in TerriaMap's `config.json` just to make it work with the way I implemented the branding stuff

Reply:
As discussed, resolved via https://github.com/TerriaJS/terriajs/pull/4830/commits/462171b89b2b6e6b53143a70974618da4100ddce